_topics/en/eclipse-attacks.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Eclipse attacks
+
+## Optional.  An entry will be added to the topics index for each alias
+#aliases:
+#  - Foo
+
+## Required.  At least one category to which this topic belongs.  See
+## schema for options
+categories:
+  - P2P Network Protocol
+  - Security Problems
+
+## Required.  Use Markdown formatting.  Only one paragraph.  No links allowed.
+## Should be less than 500 characters
+excerpt: >
+  **Eclipse attacks** occur when a node is isolated from all honest
+  peers but remains connected to at least one malicious peer.
+
+## Optional.  Use Markdown formatting.  Multiple paragraphs.  Links allowed.
+extended_summary: |
+  Without any connections to honest peers, the eclipsed node won't
+  receive the latest blocks on the most proof-of-work block chain.  This
+  gives the attacker an unlimited amount of time to generate blocks
+  containing double spends, so even an attacker with a small minority of
+  hash rate can trick the victim into accepting confirmed double spends.
+
+  The attacker will also control what transactions the victim's node
+  receives.  This allows them to tell the node about transactions not
+  generally available on the network in order to prompt the victim's
+  node into taking an action (e.g. the attacker sends a transaction
+  closing an LN channel only to the eclipsed node).  It also means that
+  any transactions generated by the victim can be definitively
+  identified as originating from the victim---a loss of privacy.
+
+  Finally, the attacker can prevent the victim from sending
+  transactions, including time-critical transactions such as LN penalty
+  transactions.
+
+  To prevent eclipse attacks, node operators are encouraged to run their
+  nodes on multiple interfaces and, when possible, to maintain
```

Comment:
Im unclear on the multiple interfaces terminology, personally.

Reply:
If it said, "multiple network interfaces", do you think that would be clear, or should I add more detail?  (I could alternatively write "multihomed" and link to https://en.wikipedia.org/wiki/Multihoming if you think that would be better)

Reply:
I like the multiple network interfaces referencing multihoming wiki as you’ve change it